src/wazuh_mcp_server/scripts/connection_validator.py

```
@@ -170,7 +170,7 @@ async def _test_https_connection(self, host: str, port: int) -> Dict:
                     return result
         except ssl.SSLCertVerificationError:
             result['self_signed'] = True
-        except Exception as e:
+        except Exception:
             result['error'] = str(e)
```

Ruff (0.11.9)

Undefined name `e`

(F821)